Python/Product/Uwp.Interpreter/PythonUwpInterpreterFactoryProvider.cs

```diff
@@ -21,106 +21,381 @@
 using System.Linq;
 using System.Reflection;
 using Microsoft.PythonTools.Interpreter;
+using Microsoft.PythonTools.Infrastructure;
```

Comment:
We'd rather you didn't use anything from this namespace, as it's supposed to be private. I can't easily see what's being used though - can you point it out for me?